### DISCUSSION TITLE
Obvious website differentiation with Nuxt 3

### DISCUSSION THREAD

**Discussion by MaelAbgrall** [nuxt/website-v2#2171] [Ideas]
Hi everyone,

I'm a beginner with Nuxt, and very often I'm looking for Nuxt 3 content, and google brings me on the version 2 of the website. Since the url and the design are very similar, I've been a few times on the wrong documentation, banging my head on why something was not working.

Would it be possible to have a more obvious way to differentiate Nuxt 2 and Nuxt 3 docs? Like with the alert and the banner at the top of every Vue 2 pages

**Comment by TheAlexLichter** [comment]
There should be a Banner regarding Nuxt 3 out there 😋

Do you feel it should be more prominent?

<img width="1468" alt="image" src="https://user-images.githubusercontent.com/640208/203847226-1b49ce69-91a3-4919-9aac-e7c21fedea1a.png">

**Reply by MaelAbgrall** [reply]
Yes, I was thinking more about the doc itself, like vue does:
![image](https://user-images.githubusercontent.com/25052677/204025285-99969c47-7531-46fe-b694-b3662a25872f.png)

**Reply by MaelAbgrall** [reply]
Happy to do a PR with all the links
